CommandLine/emake-tests/Extensions/Json/short-json-converter-test.cpp

```diff
@@ -0,0 +1,425 @@
+#include "Universal_System/Extensions/Json/short_json_converter.cpp"
+#include "Universal_System/Extensions/Json/short_json_converter.h"
+
+#include <gtest/gtest.h>
+
+#include <string>
+
+TEST(ShortJSONConverterTest, TestShortJSONSyntaxErrorInLevels_1) {
+  std::string *buffer = new std::string();
```

Comment:
I don't think its necessary to use `new` here, you can just take the address of the string and use that.